### PR TITLE
Jbanasik

### PR DESCRIPTION
**Pull request by jbanasik**
Please check if the function covers all the edge cases.

### CODE REVIEW

**Review by adpawel — APPROVED** [review]

**Comment by jbanasik** [comment]
Thanks for your approval! Highly appreciate it :D 🥇

File: Sieve.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    sieve[1] = 0
+    for i in range(0,len(sieve)):
+        if sieve[i]:
+            for j in range(i * i, len(sieve), i):
+                sieve[j] = 0
+    #print(sieve)
+    result = [x for x in range(len(sieve)) if sieve[x]]
+    return result
+
+if __name__ == "__main__":
```
Comment (endzju) [inline Sieve.py:15]: well done

Comment (endzju) [inline Sieve.py:15]: Very good

**Review by endzju — APPROVED** [review]
Good function

**Pull request merged**